boards/shields/waveshare_rs485_can/doc/index.rst

```diff
@@ -0,0 +1,122 @@
+.. _waveshare_rs485_can:
+
+WAVESHARE-RS485-CAN: RS485 CAN expansion board
+##############################################
+
+Overview
+********
+The RS485 CAN Shield will easily enable RS485/CAN communication functions for
+NUCLEO/XNUCLEO Arduino boards.
+
+ - Compatible with the ST Morpho and Arduino UNO R3
+ - RS485 function, onboard transceiver SP3485/MAX3485, 3.3V power supply
+ - CAN function, onboard transceiver SN65HVD230, 3.3V power supply
+
+.. note::
+
+   When using with Arduino UNO, Leonardo, the CAN function is unavailable due to
+   hardware limitation.
+
```

Comment:
What does this mean?

Reply:
See red marked notice in waveshare wiki: https://www.waveshare.com/wiki/RS485_CAN_Shield

Reply:
so maybe be more specific, something like the following, and upgrade from a `.. note` to a `.. warning`?

```suggestion
   The Arduino-compatible board used MUST be using 3.3V TTL logic levels for this shield to work. 

```